Sources/Datadog/FeaturesIntegration/TracingToLoggingOutput.swift

```diff
@@ -0,0 +1,44 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+/// Bridges logs created by Tracing feature to Logging feature's output. This stands for the thin integration layer
+/// between Tracing and Logging features.
+internal struct TracingToLoggingOutput {
+    /// Open Tracing standard log fields:
+    /// https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table
+    struct OpenTracingFields {
+        static let message = "message"
+        // TODO: RUMM-477 Support all standard OT log fields and expose them to the user
+    }
+
+    /// Datadog reserved log fields.
+    struct DatadogFields {
+        // TODO: RUMM-478 Add tracing log attributes to the list of reserved log attributes in `LogSanitizer`
+        static let traceID = "dd.trace_id"
+        static let spanID = "dd.span_id"
+    }
+
+    struct DefaultFieldValues {
+        static let message = "Span event"
+    }
+
+    // MARK: - TraceLogOutput
+
+    /// `LogOutput` provided by the `Logging` feature.
+    let loggingOutput: LogOutput
+
+    func writeLogWith(spanContext: DDSpanContext, fields: [String: Encodable], date: Date) {
```

Comment:
this method name doesn't seem to follow Swift naming conventions 
(doesn't the compiler complain about it?)
```suggestion
    func writeLog(with spanContext: DDSpanContext, fields: [String: Encodable], date: Date)
```

Reply:
Done 👌. You mean the linter, not compiler warning? No, swiftlint [doesn't define any rules about method naming](https://realm.github.io/SwiftLint/rule-directory.html). I think such a rule would give too many false positives.